Search feat

Alejo-FM opened the pull request:
Se realizo todo el search, pero se hace PR para poder obtener los cambios relacionados al SafeBloc y los test para aprovechar y hacerlos.

Alejo-FM committed:
feat: :sparkles: Bloc Search & Tags

Se añadieron 2 blocs, uno para manejrar la busqueda y otro para cargar los tags al inicio

Alejo-FM committed:
feat: :sparkles: Datasource & Repository Domain Search

Se añadieron las interfaces para todo lo relacionado con la busqueda

Alejo-FM committed:
feat: :sparkles: Implementation Datasource & Repository Search

Se realizo la logica en infrastructura para el funcionamiento del search

Alejo-FM committed:
perf: :zap: Improve

Se añadio la configuración de las notificaciones para iphone y se modifico los models de blog y courses

Alejo-FM committed:
feat: :sparkles: Search Screens & Widgets related

Se realizo la screen de search a espera de una mejora luego de hacer merge, ademas se añadio el import en screens

Alejo-FM committed:
style: :lipstick: Improve style

En login hice un leve movimiento para que la palabra login no estuviera pegada al borde, en el appbar añadi el push para ir a la scrren y en blogs_slideshow pequeños cambios visuales

Alejo-FM committed:
style: :lipstick: Ultimos cambios

Se añadio la ruta del screen, se hizo el courses_slidesho y el customapp bar un cambio pequeño

Alejo-FM committed:
Merge branch 'development' into search-feat

GianL22 commented on lib/infrastructure/datasources/search/api_search_datasource.dart:
haz el cambio del auth de una vez

Alejo-FM replied:
Listo :D

Alejo-FM committed:
fix: :bug: Bearer Token Change

Se cambio el header de auth a Bearer token en base a los nuevos cambios del back

Alejo-FM committed:
Merge branch 'search-feat' of https://github.com/DeltaTeam-UCAB/gymnastic-center-mobile into search-feat
